openshift/internal-services-controller/deployment.yaml

```diff
@@ -76,7 +76,7 @@ spec:
             - /mnt/internal-services/remote-client-config/kubeconfig
           command:
             - /manager
-          image: quay.io/redhat-appstudio/internal-services:fdb328071717d4136fec5144c06c77893b0750cd
+          image: quay.io/redhat-appstudio/internal-services:5d590262c067ee78acbb8c410accd1f99af712ac
```

Comment:
The InternalServicesConfig CRD hast to be added. The Internal Request CRD needs to be updated as well. Are you doing it as part of this PR or you want me to do it in a separate one?

Reply:
I will do as part of this PR. Thanks for catching this